docs/sql/statements/alter-table.rst

```diff
@@ -175,11 +175,11 @@ expression.
 ``OPEN/CLOSE``
 --------------
 
-Can be used to open or close the table, respectively. Closing a table prevents
-all operations, except ``ALTER TABLE ... OPEN``, to fail. Operations on closed
-partitions will not produce an exception, but will have no effect. Similarly,
-like ``SELECT`` and ``INSERT`` on partitioned will exclude closed partitions and
-continue working.
+Can be used to open or close the table.
+
+Closing a table means that all operations, except ``ALTER TABLE ... OPEN``,
+will fail. Operations that fail will not return an error, but they will have no
```

Comment:
Something like that?
The difference here is: operating on partitioned tables containing closed partitions vs. operating directly on closed tables or closed partitions.

```suggestion
will fail. Operations on tables containing closed partitions, won't fail but will exclude all closed partitions from the actual operation.
```